bazel_tools/scala/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@io_bazel_rules_scala//scala:scala_toolchain.bzl", "scala_toolchain")
+
+scala_toolchain(
+    name = "toolchain_impl",
+    enable_stats_file = False,
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "toolchain",
+    toolchain = "toolchain_impl",
+    toolchain_type = "@io_bazel_rules_scala//scala:toolchain_type",
+    visibility = ["//visibility:public"],
+)
```

Comment:
This is the same as the [default Scala toolchain](https://github.com/bazelbuild/rules_scala/blob/17791a18aa966cdf2babb004822e6c70a7decc76/scala/BUILD#L10-L21) except for `enable_stats_file = False`.